.travis.yml

```diff
@@ -18,15 +18,16 @@ addons:
     - g++-4.8
     - python3.5
 cache:
+  yarn: true
   directories:
   - node_modules
-before_script:
+before_install:
 - curl -fsSL https://bootstrap.pypa.io/get-pip.py | python3.5 - --user
 - travis_retry pip3.5 install --user transifex-client
 - install -m0644 .transifexrc.tpl ~/.transifexrc
 - echo "password = $TX_PASSWD" >> ~/.transifexrc
 script:
-- npm run build
-- npm run test
+- yarn test
+- yarn run build
```

Comment:
Hmm, yep, because we don't test the build, but the sources. So fast fail when tests are KO, before triggering the build seems legit, nope?